libg933/src/lib.rs

```diff
@@ -255,7 +255,8 @@ impl Device {
         let length = self.raw_request(&request)?[4];
 
         let mut name = String::new();
-        for i in 0..length / 10 {
+        let name_parts = ((length - 1) / 10) + 1;
```

Comment:
What is this? It gets the wrong number of parts for the G933, returning 4 instead of 3

Reply:
I had my name chopped of, because length/10 was rounding it down. What is the device name of G933 headset?

Reply:
The device name is 25 characters long, it's in the notes

Reply:
oops, it's actually 37 characters long, because hex. but it's also 16 characters returned in the message...

whoops...

Reply:
Just change the 10 to a 16 and it should be fine :+1: